app/dedicated-cloud-gateways/custom-dns.md

```diff
@@ -0,0 +1,107 @@
+      3. Wait for DNS propagation and try attaching your domain again.
+
+      If no CAA record exists, no changes are needed. For more details, see the [Let's Encrypt CAA Guide](https://letsencrypt.org/docs/caa/).
+
+  - q: How do I configure a custom domain in {{site.konnect_short_name}}?
+    a: |
+      In {{site.konnect_short_name}}, go to [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), select a Control Plane, open the **Overview** dashboard, click **Connect**, and save the **Public Edge DNS** URL. 
+      Then, navigate to **Custom Domains**, click **New Custom Domain**, enter your domain name, and save the CNAME and Content values.
+
+  - q: What DNS records do I need to add at my domain registrar for a custom domain?
+    a: |
+      You need to create CNAME records using the values provided in {{site.konnect_short_name}}.
+      * `_acme-challenge.example.com` → `_acme-challenge.9e454bcfec.acme.gateways.konghq.com`
+      * `example.com` → `9e454bcfec.gateways.konghq.com`
+
+  - q: How often is DNS validation refreshed for Dedicated Cloud Gateways?
+    a: |
+      DNS validation statuses for Dedicated Cloud Gateways are refreshed every 5 minutes.
+  
+  - q: How do I delete a custom domain in {{site.konnect_short_name}}?
+    a: |
+      In {{site.konnect_short_name}}, go to [**Gateway Manager**](https://cloud.konghq.com/us/gateway-manager/), choose a Control Plane, click **Custom Domains**, and use the action menu to delete the domain.
+
+  - q: Why is my custom domain attachment failing in {{site.konnect_short_name}}?
+    a: |
+      A common reason is a missing or misconfigured Certificate Authority Authorization (CAA) record. 
+      {{site.konnect_short_name}} uses Google Cloud Public CA (`pki.goog`) to issue certificates. 
+      If your domain's CAA record does not authorize this CA, attachment will fail.
+
+  - q: How can I fix CAA record issues for a custom domain?
+    a: |
+      Use `dig CAA yourdomain.com +short` to check for existing CAA records. 
+      If needed, update the record to allow `pki.goog`, like this,
+      `yourdomain.com.    CAA    0 issue "pki.goog"`  
+      Then, wait for DNS propagation and retry the domain attachment.
+
+related_resources:
+  - text: Konnect Advanced Analytics
+    url: /advanced-analytics/
+---
+
+
+{{site.konnect_short_name}} integrates domain name management and configuration with [Dedicated Cloud Gateways](/dedicated-cloud-gateways/).
+
+## {{site.konnect_short_name}} configuration
+
+1. Open **Gateway Manager**, choose a control plane to open the **Overview** dashboard, then click **Connect**.
+    
+    The **Connect** menu will open and display the URL for the **Public Edge DNS**. Save this URL.
+
+1. Select **Custom Domains** from the side navigation, then **New Custom Domain**, and enter your domain name.
+
+    Save the value that appears under **CNAME**. 
+
+## Dedicated Cloud Gateways domain registrar configuration
+
+To configure 
+{% table %}
```

Comment:
There's a an issue with the table width ![image](https://github.com/user-attachments/assets/84d65400-2f58-482a-8198-7e031bdb167b)